locale/lt/LC_MESSAGES/crm.po

```diff
@@ -1012,6 +1019,88 @@ msgstr ""
 "naujas praradimo pajamas, jei yra poreikis jas atnaujinti, atliekant šį "
 "veiksmą."
 
+#: ../../crm/customizations/mail_activity_systray_for_crm.rst:3
+msgid "Mail Activity Systray for CRM"
+msgstr "CRM suplanuotų veiklų panelė"
+
+#: ../../crm/customizations/mail_activity_systray_for_crm.rst:5
```

Comment:
Dont forget to update translations, because you were using incorrect `mail_activity_systray_for_crm` name here.

Reply:
Translations updated